### PR TITLE
Match benchmark iOS project bundle identifiers

### PR DESCRIPTION
**Pull request by jmagman**
## Description

Match benchmark Info.plist bundle identifiers to their project settings.  Don't reuse complex layout ID.

### CODE REVIEW

File: dev/benchmarks/platform_views_layout/ios/Runner/Info.plist [hunk 1]
```diff
@@ -44,6 +44,6 @@
 	<key>io.flutter.embedded_views_preview</key>
 	<true/>
 	<key>io.flutter.metal_preview</key>
- 	<true/>
+	<true/>
```
Comment (jmagman) [inline dev/benchmarks/platform_views_layout/ios/Runner/Info.plist:47]: Xcode change this whitespace, let it.

File: dev/benchmarks/platform_views_layout/ios/Runner/Info.plist [hunk 2]
```diff
@@ -7,7 +7,7 @@
 	<key>CFBundleExecutable</key>
 	<string>$(EXECUTABLE_NAME)</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.yourcompany.complexLayout</string>
+	<string>com.yourcompany.platformViewsLayout</string>
```
Comment (jmagman) [inline dev/benchmarks/platform_views_layout/ios/Runner/Info.plist:10]: Match https://github.com/flutter/flutter/blob/2e67bf2e32391af4fe38cb292c99e77d14b53584/dev/benchmarks/platform_views_layout/ios/Runner.xcodeproj/project.pbxproj#L319

File: dev/benchmarks/platform_views_layout_hybrid_composition/ios/Runner/Info.plist [hunk 3]
```diff
@@ -7,7 +7,7 @@
 	<key>CFBundleExecutable</key>
 	<string>$(EXECUTABLE_NAME)</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.yourcompany.complexLayout</string>
+	<string>com.yourcompany.platformViewsLayoutHybridComposition</string>
```
Comment (jmagman) [inline dev/benchmarks/platform_views_layout_hybrid_composition/ios/Runner/Info.plist:10]: Match https://github.com/flutter/flutter/blob/2e67bf2e32391af4fe38cb292c99e77d14b53584/dev/benchmarks/platform_views_layout_hybrid_composition/ios/Runner.xcodeproj/project.pbxproj#L319

**Review by christopherfujino — APPROVED** [review]
LGTM

**Comment by jmagman** [comment]
Not confident this will fix https://github.com/flutter/flutter/issues/65734, but landing on red to see!

**Pull request merged**